scripts/getBenchmarkVersion.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '[INFO]'
```

Comment:
Can you update this script to also return the Benchmark version it found to the console. That way you can test/use it independently to verify it works.

Reply:
What do you mean by `return the Benchmark version it found to the console`? If you run the script directly, it outputs `1.2`.

Reply:
Duh. Nevermind. Although there is a 1st run problem where when you run it the 1st time, all the output generated by downloading the plugin gets included in the filename.  I think that can be fixed by including that plugin in the pom.xml so its already downloaded by the time any script using this plugin runs.

Reply:
I just run the check twice and forward the first run to /dev/null.